### DISCUSSION TITLE
Dropdown Color

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#1765] [Q&A] [answered]
Hey,

is it possible to set the clicked color in a Dropdown?
Because if I set my primary to black it does not look good.

![image](https://user-images.githubusercontent.com/10349466/106269746-33615800-622d-11eb-83db-fbe7de14d4ea.png)

**Comment by stsrki** [comment]
How do you set the colors? Can you show the code?

**Reply by bastigerner** [reply]
Export Button Color is  `<DropdownToggle Color="Color.Secondary">`
and my Primary Color is black set in the Theme via 

```
ColorOptions = new ThemeColorOptions
                    {
                        Primary = color,
```

**Comment by stsrki** [comment]
So the problem here is the text color is not reconfigured to be `white`?

**Reply by bastigerner** [reply]
Yes this is the problem. 
Either the text needs to be white or the clicked color can be the primary color but not with 100% opacity

**Comment by stsrki** [comment, marked as answer]
The fix should be easy to do. Can you open a new issue and I will fix it for the next release?

In the meantime try this

```css
.dropdown-item.active, .dropdown-item:active{
  color: white;
}
```

**Reply by bastigerner** [reply]
Okay added it like this but is still black on press

![image](https://user-images.githubusercontent.com/10349466/106272028-ab7d4d00-6230-11eb-9f8d-6d64e75b76ba.png)

**Comment by stsrki** [comment]
It's probably because theme styles still override them.

Try adding `!important` flag,

**Reply by bastigerner** [reply]
Okay good it worked! Thanks

**Comment by bastigerner** [comment]
Created a Issue

**Reply by stsrki** [reply]
Thanks
